Library/ENSideMenu.swift

```diff
@@ -334,6 +343,7 @@ open class ENSideMenu : NSObject, UIGestureRecognizerDelegate {
                 withDuration: animationDuration,
                 animations: { [weak self] () -> Void in
                     self?.sideMenuContainerView.frame = destFrame
+                    self?.delegate?.sideMenuIsAnimating()
```

Comment:
I anticipate some unexpected side effects on the use site if the delegate method would be called inside an animation block. Instead I would suggest introducing another two delegate callbacks with an `isAnimated` argument and call them before scheduling the animation:
`sideMenuWillOpen:isAnimated`
 `sideMenuWillClose:isAnimated`
I see this as a more universal approach.

Reply:
This is a good point, but my specific case requries the delegate call inside the animate method so the statubar changes gradually not at once wzen animation startes or finishes.